databricks/koalas/indexes.py

```diff
@@ -3318,6 +3319,40 @@ def item(self) -> Tuple[Scalar, ...]:
         """
         return self._kdf.head(2)._to_internal_pandas().index.item()
 
+    def equal_levels(self, other):
+        """
+        Return True if the levels of both MultiIndex objects are the same
+
+        Examples
+        --------
+        >>> from databricks.koalas.config import set_option, reset_option
+        >>> set_option("compute.ops_on_diff_frames", True)
+
+        >>> kmidx1 = ks.MultiIndex.from_tuples([("a", "x"), ("b", "y"), ("c", "z")])
+        >>> kmidx2 = ks.MultiIndex.from_tuples([("b", "y"), ("a", "x"), ("c", "z")])
+        >>> kmidx1.equal_levels(kmidx2)
+        True
+
+        >>> kmidx2 = ks.MultiIndex.from_tuples([("a", "x"), ("b", "y"), ("c", "j")])
+        >>> kmidx1.equal_levels(kmidx2)
+        False
+
+        >>> reset_option("compute.ops_on_diff_frames")
+        """
+        nlevels = self.nlevels
+        if nlevels != other.nlevels:
+            return False
+        self_frame = self.sort_values().to_frame()
+        other_frame = other.sort_values().to_frame()
+        with option_context("compute.ops_on_diff_frames", True):
```

Comment:
We might avoid force enabling `compute.ops_on_diff_frames`. let's see.

Reply:
Okay, thanks for the review! Let me resolve the comments